auth/consts.go

```diff
@@ -3,18 +3,10 @@
 
 package auth
 
-import "github.com/ava-labs/hypersdk/vm"
-
 // Note: Registry will error during initialization if a duplicate ID is assigned. We explicitly assign IDs to avoid accidental remapping.
 const (
 	// Auth TypeIDs
 	ED25519ID   uint8 = 0
 	SECP256R1ID uint8 = 1
 	BLSID       uint8 = 2
 )
-
-func Engines() map[uint8]vm.AuthEngine {
-	return map[uint8]vm.AuthEngine{
-		ED25519ID: &ED25519AuthEngine{},
-	}
-}
```

Comment:
This function was unused and now creates an import cycle. It was also probably not correctly placed.

Reply:
Oh actually it's called `examples/morpheusvm/controller/controller.go`, let's find a better place

Reply:
Moved it to `auth/common`, let me know if it's a good choice.